dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/CoreFeature.kt

```diff
@@ -99,11 +102,18 @@ internal object CoreFeature {
             systemInfoProvider = NoOpSystemInfoProvider()
             networkInfoProvider = NoOpNetworkInfoProvider()
             userInfoProvider = NoOpMutableUserInfoProvider()
+            serviceName = ""
+            packageName = ""
+            packageVersion = ""
             shutDownExecutors()
             initialized.set(false)
         }
     }
 
+    fun resolveServiceName(): String {
+        return serviceName ?: packageName
+    }
+
```

Comment:
You can delete this one, it's not used anymore

Reply:
Yup..again forgot, damn it.